docs/content/v2.0/admin/yb-admin.md

```diff
@@ -534,18 +534,23 @@ yb-admin -master_addresses <master-addresses> modify_placement_info <placement_i
 ```
```

Comment:
@NitronR this is the wrong file to make docs changes. I had commented to make changes in https://github.com/yugabyte/yugabyte-db/blob/master/docs/content/latest/admin/yb-admin.md (i.e. the latest version) and not on v2.0

Reply:
Oh right, missed that. Thanks for pointing that out.